deploy/environments/externals.yaml

```diff
@@ -51,6 +51,7 @@ spack:
     - hpctoolkit@2020.03.01
     - hpcviewer
     - hpe-mpi@2.22.hmpt
+    - hypre+int64+mpi
```

Comment:
@tristan0x : Is It better to have this in libraries than external? reason is that all dependencies of "external" packages is trimmed.